src/main/java/com/example/UserService.java

```diff
@@ -0,0 +1,29 @@
+package com.example;
+
+import java.util.List;
+
+public class userService {
+    private List<String> users;
+
+    // Constructor injection
+    public UserService(List<String> users) {
+        this.users = users;
+    }
+
+    // Rest of the code...
+
+    public boolean isUserExists(String userName) {
+        boolean f = false;
+
+        for (int index = 0; index < users.size(); index++) {
+            String user = users.get(index);
```

Comment:
You could use an enhanced for loop (e.g. `for (String user : users)`)

Reply:
Could use users.contains